crates/torii/migrations/20230316154230_setup.sql

```diff
@@ -10,14 +10,13 @@ CREATE TABLE worlds (
     world_class_hash TEXT,
     executor_address TEXT,
     executor_class_hash TEXT,
-    created_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
+    executed_at DATETIME NOT NULL DEFAULT CURRENT_TIMESTAMP
```

Comment:
I don't think we want a default for `executed_at`

Reply:
Yeah I'm going slowly with the updates, tried to make everything in once but for some reasons I have different results when running the test suite locally & the CI so I'm going step by step - I should probably setup a docker container